src/map/utils/battleutils.cpp

```diff
@@ -2020,7 +2020,7 @@ namespace battleutils
             if (giveTPtoVictim)
             {
                 //account for attacker's subtle blow which reduces the baseTP gain for the defender
-                float sBlowMult = ((100.0f - dsp_cap((float)PAttacker->getMod(Mod::SUBTLE_BLOW), 0.0f, 50.0f)) / 100.0f);
+                float sBlowMult = ((100.0f - (dsp_cap((float)PAttacker->getMod(Mod::SUBTLE_BLOW), 0.0f, 50.0f) + dsp_cap((float)PAttacker->getMod(Mod::SUBTLE_BLOW_II), 0.0f, 25.0f))) / 100.0f);
```

Comment:
Regarding Cure Potency II, the info I found seems to suggest that there's no cap so I'll be removing that check. 
`Equipment that has "Cure potency II" as a stat will increase beyond the 50% cap. `

There is also another known cap that's not currently being enforced but adding that to all the healing spells shouldn't be too much work.
`The casted and received potency caps are separate and multiply with each other for a maximum possible potency multiplier of 1.95 (1.5 x 1.3).`